discord/client.py

```diff
@@ -541,6 +543,38 @@ async def on_error(self, event_method: str, *args: Any, **kwargs: Any) -> None:
         print(f"Ignoring exception in {event_method}", file=sys.stderr)
         traceback.print_exc()
 
+    async def on_view_error(
+        self, error: Exception, item: Item, interaction: Interaction
+    ) -> None:
+        """|coro|
+
+        The default view error handler provided by the client.
+
+        This only fires for a view if you did not define its :func:`~discord.ui.View.on_error`.
+        """
+
+        print(
```

Comment:
Should consider using something else than `print`

Reply:
I agree but this is copying the other error methods in the library; I think it would be appropriate to start a separate PR that addresses usage of print across the entire library rather than deciding it here